/**:
  ros__parameters:  
    crazyflieTypes:
      default: 
        markerConfiguration: 4
        dynamicsConfiguration: 0